roles/grafana/templates/dashboards.yaml.j2

```diff
@@ -0,0 +1,9 @@
+apiVersion: 1
+
+providers:
+  - name: 'Dashboards'
+    updateIntervalSeconds: 10
+    allowUiUpdates: false
+    options:
+      path: /var/lib/grafana/dashboards/
```

Comment:
```suggestion
      path: {{ grafana__dashboards_path | to_yaml }}
```

Reply:
I omitted `| to_yaml` part:
```
path: {{ grafana__dashboards_path }}
```
and it works... Is that ok?

Reply:
that is to avoid breaking yaml parsing if the file name contains weird but valid characters (e.g. `[my directory with brakets]` or `dir: name`), it is just me being extra careful :)